assets/js/src/newsletters/types.jsx

```diff
@@ -57,6 +58,67 @@ class NewsletterTypes extends React.Component {
     });
   };
 
+  getAdditionalTypes = () => {
+    const show = window.mailpoet_woocommerce_active && MailPoet.FeaturesController.isSupported('wc-transactional-emails-customizer');
+    if (!show) {
+      return [];
+    }
+    return [
+      {
+        slug: 'wc_transactional',
+        title: MailPoet.I18n.t('wooCommerceCustomizerTypeTitle'),
+        description: MailPoet.I18n.t('wooCommerceCustomizerTypeDescription'),
+        action: (
+          <a
+            className="button button-primary"
+            data-automation-id="customize_woocommerce"
+            onClick={this.openWooCommerceCustomizer}
+            role="button"
+            tabIndex={0}
+            onKeyDown={(event) => {
+              if ((['keydown', 'keypress'].includes(event.type) && ['Enter', ' '].includes(event.key))
+              ) {
+                event.preventDefault();
+                this.openWooCommerceCustomizer();
+              }
+            }}
+          >
+            {MailPoet.I18n.t(window.mailpoet_woocommerce_customizer_enabled ? 'customize' : 'activate_and_customize')}
+          </a>
+        ),
+      },
+    ];
+  };
+
+  openWooCommerceCustomizer = () => {
+    const promise = jQuery.Deferred();
+    MailPoet.trackEvent('Emails > Type selected', {
+      'MailPoet Free version': window.mailpoet_version,
+      'Email type': 'wc_transactional',
+    });
+    if (!window.mailpoet_woocommerce_customizer_enabled) {
+      promise.then(() => MailPoet.Ajax.post({
+        api_version: window.mailpoet_api_version,
+        endpoint: 'settings',
+        action: 'set',
+        data: {
+          'woocommerce.use_mailpoet_editor': true,
```

Comment:
I think we should use `1` here otherwise it will be saved as string `"true"`. And if someone saw the code in the future and would like to set it to false, having string `"false"` would still be truthy.

Reply:
You're right.